molgenis-data-annotators/src/main/java/org/molgenis/data/annotator/tabix/TabixRepository.java

```diff
@@ -92,7 +99,11 @@ public EntityMetaData getEntityMetaData()
 			String line = iterator.next();
 			while (line != null)
 			{
-				builder.add(toEntity(line));
+				Entity entity = toEntity(line);
+				if (entity.getLong(VcfRepository.POS) == pos)
```

Comment:
I checked the tabix documentation and you can specify the index of the pos column. So it might be something else than "POS".
However, the tabixreader we use doesn't have a function to return which column it is as far as I can tell... so no better alternative available.